### DISCUSSION TITLE
Ajout du réseau mastodon

### DISCUSSION THREAD

**Discussion by ghost** [AmauriC/tarteaucitron.js#1111] [Ideas]
Bonsoir,

De plus en plus d'internautes quittent Twitter pour aller sur le réseau social mastodon.
Est il possible d'envisager de l'intégrer dans Tarteaucitron
Au plaisir de vous suivre
Linuxmr

**Comment by AmauriC** [comment]
Bien sûr, je peux l'ajouter :)

Il me faut juste le marqueur html nécessaire pour intégrer Mastodon sur un site pour faire l'équivalent tarteaucitron.

**Reply by AmauriC** [reply]
Non, il me faut le code html/javascript que Mastodon demande d'ajouter sur le site pour intégrer le contenu Mastodon sur un site.

**Reply by AmauriC** [reply]
Tarteaucitron ne gère que la partie Javascript, pas côté serveur.

Pour que j'ajoutes le service Mastodon il me faut le marqueur html/js qu'il faut ajouter sur le site.
